handsontable/src/plugins/contextMenu/menu.js

```diff
@@ -432,10 +447,20 @@ class Menu {
       } else {
         this.setPositionBelowCursor(cursor);
       }
-      if (cursor.fitsOnRight(this.container)) {
-        this.setPositionOnRightOfCursor(cursor);
-      } else {
-        this.setPositionOnLeftOfCursor(cursor);
+
+      if (this.hot.isLtr()) { // ltr mode
+        if (cursor.fitsOnRight(this.container)) {
+          this.setPositionOnRightOfCursor(cursor);
+        } else {
```

Comment:
Core Handsontable team use extra space in such conditionals. I am aware you may not have known this. However, it's hard to force this style by linter.

```suggestion
          this.setPositionOnRightOfCursor(cursor);

        } else {
```

Reply:
IMHO, this is task for eslint

Reply:
Please do not close comments before resolving.